Programs/_testembed.c

```diff
@@ -343,12 +343,9 @@ static int test_pre_initialization_sys_options(void)
 
     _Py_EMBED_PREINIT_CHECK("Checking PySys_AddWarnOption\n");
     PySys_AddWarnOption(L"default");
-    _Py_EMBED_PREINIT_CHECK("Checking PySys_ResetWarnOptions\n");
```

Comment:
Since it's still part of the stable ABI, you can keep it in the test. You just need to declare it.

It's good to test the stable ABI.